internal/vm-console-proxy-bundle/bundle.go

```diff
@@ -105,7 +114,7 @@ func loadBundleFromBytes(data []byte) (*Bundle, error) {
 }
 
 func validateBundle(bundle *Bundle) error {
-	missingFields := make([]string, 0, 6)
+	missingFields := make([]string, 0, 7)
```

Comment:
If you want to allocate space for all checks below this should be `8`?

Reply:
Thanks, I missed it.